example/topics/main.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
```

Comment:
Please change `2017` to `2019`. We place the current year in each file as it is created. Thanks.

Reply:
done